### ISSUE TITLE
testcode

### ISSUE THREAD

**Issue by siwol37** [schoolhaterclub/sendice#1]
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
    integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
      integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
      crossorigin="anonymous"></script>
      <link href="https://fonts.googleapis.com/css?family=Noto+Sans+KR:400,700&amp;subset=korean" rel="stylesheet">
    <title>Sendice</title>

    <script>
        function enter() {
            alert ('hello')    
        }
    </script>

    <style>
        *{
            font-family: 'Noto Sans KR', sans-serif;

        }

        .main {
            display: flex;
            flex-direction: column;
            justify-content: center;
            align-items: center;

            background-image: url(https://cdn.discordapp.com/attachments/873159653704470551/1011946283931533342/unknown.png);
            background-position: center;
            background-size: cover;
        }

        .img {
            height: 100px;
            width: 100px;
            border-radius: 500px;
            
            margin: auto;
            margin-top: 12.5px;
            margin-left: 12.5px;
            padding: auto;

        }
        .title {

            font-size: 25px;

            margin: auto;
            margin-top: 50px;

            display: flex;
            flex-direction: column;
            justify-content: center;
            align-items: center;
        }

        .subtitle {
            font-size: 20px;

            margin: auto;

            display: flex;
            flex-direction: column;
            justify-content: center;
            align-items: center;
        }

        .title-oneblock {
            height: 250px;
        }

        .title-button {
            margin-left: 500px;
            margin-top: 30px;
            margin-bottom: 15px;
            
            font-size: 20px;

            width: 150px;
            height: 50px;
            
            color: white;
            border: transparent;

            background: #43cea2; /* fallback for old browsers */
            background: -webkit-linear-gradient(to right, #43cea2, #185a9d); /* Chrome 10-25, Safari 5.1-6 */
            background: linear-gradient(to right, #43cea2, #185a9d); /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */

        }
        .button-test {
            background: #2196F3;  /* fallback for old browsers */
            background: -webkit-linear-gradient(to right, #F44336, #2196F3);  /* Chrome 10-25, Safari 5.1-6 */
            background: linear-gradient(to right, #F44336, #2196F3); /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
            font-size: 20px;
            width: 300px;
            height: 100px;
            border-radius: 500px;
            border-color: white;
            border: 0.2px;
        }
        .button-test:hover {
            background: #2196F3;  /* fallback for old browsers */
            background: -webkit-linear-gradient(to right, #F44336, #2196F3);  /* Chrome 10-25, Safari 5.1-6 */
            background: linear-gradient(to right, #F44336, #2196F3); /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
            font-size: 21px;
            width: 300px;
            height: 100px;
            padding: auto;
            border-radius: 500px;
        }
        .form-control {
            width: 500px;
            height: 50px;

        }

        .search-logo:hover {

            background: #d6d8da;  /* fallback for old browsers */

            color: white;
            
        }
        
        
        
        .search {
            margin: 100px auto 200px auto;
        }

        #searchbar {
            margin-top: 50px;
            height: 50px;
            margin: auto;

            float: left;

        }

        .search-logo {
            width: 50px;
            height: 50px;
            border-radius: 500px;

            float: left;

            margin-left: 20px;                 
            background-color: white;

            border-image: #2196F3;  /* fallback for old browsers */
            border-image: -webkit-linear-gradient(to right, #F44336, #2196F3);  /* Chrome 10-25, Safari 5.1-6 */
            border-image: linear-gradient(to right, #F44336, #2196F3); /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
            border-image-slice: 1; 
            border: 1px solid;
        }

        .topbar {
            width: 1200px;
            height: 125px;

            background-color: white;

            margin: 0 auto 0 auto;

            }

        .topbar-button {
            display: flex;
            flex-direction: column;
            justify-content: center;
            align-items: center;

            padding: 39.5px;

            float: right;
            
            font-size: 20px;

            text-decoration-line: none;
            color: black;
        }

        /* .topbar-button-right {
            padding: auto;
            
            background-color: green;

            float: right;
            margin-top: 45px;
            margin-right: 30px;

            
            font-size: 20px;

            text-decoration-line: none;
            color: black;

            margin-left: 30px;
        } */

        .topbar-button-line {
            text-decoration-line: none;
            color: black;
        }

        .topbar-button:hover {
            display: flex;
            flex-direction: column;
            justify-content: center;
            align-items: center;

            padding: 39.5px;

            float: right;
            
            font-size: 20px;

            text-decoration-line: none;
            color: white;

            background: #43cea2; /* fallback for old browsers */
            background: -webkit-linear-gradient(to right, #43cea2, #185a9d); /* Chrome 10-25, Safari 5.1-6 */
            background: linear-gradient(to right, #43cea2, #185a9d); /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
        }

        .container-1200 {
            text-align: right;

            height: 300px;
            width: 1200px;

            margin: auto;
            margin-top: 20px;

            background-color: rgb(240, 240, 240);

            padding-top: 70px;
            padding-right: 15px;
            
        }
        .container-1200-2 {
            text-align: left;

            height: 300px;
            width: 1200px;

            margin: auto;
            margin-top: 20px;

            background-color: rgb(250, 248, 248);

            padding-top: 70px;
            padding-right: 15px;

            text-align: l;

        }

        .container-1200-h3 {

            color: #378CFF;

            font-size: 25px;
            font-weight: bold;

            padding-top: auto;
        }

        .container-1200-b {

            color: black;

            font-size: 32px;

        }

        .container-1200-p {
            margin-bottom: 0px;

            color: black;

            font-size: 16px;
        }

        /* .container-1200-content {
            text-align: right;
        } */
        
        .container-1200-p-more {

            color: #378CFF;

            font-size: 16px;

            
        }

        .last-h1 {
            margin-top: 100px;
            text-align: center;
            font-weight: bold;
        }

        .last-button {
            background-color: green;
            
            margin-left: 850px;
            margin-top: 30px;

            font-size: 20px;

            width: 200px;
            height: 50px;
            
            color: white;
            border: transparent;

            background: #43cea2; /* fallback for old browsers */
            background: -webkit-linear-gradient(to right, #43cea2, #185a9d); /* Chrome 10-25, Safari 5.1-6 */
            background: linear-gradient(to right, #43cea2, #185a9d); /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */

            margin-bottom: 100px;

        }

        .last-p {
            margin-bottom: 0px;

            color: black;
            text-align: center;

            font-size: 16px;
        }

        .last-box {
            background-color: #141e28;

            width: 100%;
            height: 200px;
            
        }

        .last-box-h3 {
            text-align: center;

            font-size: 20px;

            color: white;

            margin-top: 90px;
        }
    </style>
</head>
<body>
    <div class="main">                          
            <div class="title-oneblock">
                <div class="topbar">
                    <a href="http://sendice.xyz/main"><img src="" class="img"></a>
                    <a href="http://sendice.xyz/profile" target="_blank" class="topbar-button"><p>개발자 프로필</p></a>
                    <a href="http://sendice.xyz/test" target="_blank" class="topbar-button"><p>테스트</p></a>
                    <a href="http://sendice.xyz/corp" target="_blank" class="topbar-button"><p>회사 소개</p></a>
                    <a href="https://sendice.xyz" target="_blank" class="topbar-button"><p>메인</p></a>
                </div>
                <b class="title">지금까지 없던 네트워크 서비스를 제공합니다.</b>
                <p class="subtitle">Sendice의 대용량 네트워크 백본과 서버를 이용하여 여러분의 서비스를 창조해 나갑니다.</p>
                <a href="http://sendice.xyz" target="_blank"><button class="title-button">서비스 지원</button></a>
            </div>
            <div class="search">
                    <input id="searchbar" class="form-control" type="text" placeholder="Search" aria-label="nice" onkeyup="if(window.event.keyCode==13){enter()}">
                    <a href="https://google.com" target="_blank"><img src="https://cdn.discordapp.com/attachments/873159653704470551/1011931111477219360/517a2928209d39bb.png" class="search-logo"></a>
            </div>
    </div>
    <div class="container-1200">
        <div class="container-1200-content">
            <h3 class="container-1200-h3">Automated</h3>
            <b class="container-1200-b">상담 요청 없이 손쉽게 생성/관리 하세요.</b>
            <p class="container-1200-p">Stella IT에서 자체 개발 및 유지보수 하는 기술을 바탕으로 한</p>
            <p class="container-1200-p">Stella IT Console과 Stella API Platform을 소개합니다.</p>
            <a href="https://google.com" target="_blank"><p class="container-1200-p-more">더 알아보기</p></a>
        </div>
    </div>
    <div class="container-1200-2">
        <div class="container-1200-content">
            <h3 class="container-1200-h3">BlueText</h3>
            <b class="container-1200-b">상담 요청 없이 손쉽게 생성/관리 하세요.</b>
            <p class="container-1200-p">Stella IT에서 자체 개발 및 유지보수 하는 기술을 바탕으로 한</p>
            <p class="container-1200-p">Stella IT Console과 Stella API Platform을 소개합니다.</p>
            <a href="https://google.com" target="_blank"><p class="container-1200-p-more">더 알아보기</p></a>
        </div>
    </div>
    <div class="container-1200">
        <div class="container-1200-content">
            <h3 class="container-1200-h3">BlueText</h3>
            <b class="container-1200-b">상담 요청 없이 손쉽게 생성/관리 하세요.</b>
            <p class="container-1200-p">Stella IT에서 자체 개발 및 유지보수 하는 기술을 바탕으로 한</p>
            <p class="container-1200-p">Stella IT Console과 Stella API Platform을 소개합니다.</p>
            <a href="https://google.com" target="_blank"><p class="container-1200-p-more">더 알아보기</p></a>
        </div>
    </div>
        <h1 class="last-h1">더 궁금하신 점이 있으신가요?</h1>
        <p class="last-p">Stella IT의 전문 상담원이 고객님께 적합한 상품을 제안해 드립니다.</p>
        <a href="http://sendice.xyz" target="_blank"><button class="last-button">지금 바로 상담하기</button></a>
    <div class="last-box">
        <h3 class="last-box-h3">순순히 어두운 밤을 받아들이지 마오. 노인들이여 저무는 하루에 소리치고 저항해요. 분노하고 분노해요. 사라져가는 빛에 대해.</h3>
    </div>
</body>
</html>
